code/clue.js

```diff
@@ -1,59 +1,237 @@
+
+const rooms = [
+  MonicaDiningRoom,
+  RossApartment,
+  ChandlerDiningRoom, 
+  RachelRoom, 
+  MonicaRoom, 
+  MonicaLivingRoom, 
+  CentralPerk, 
+  Balcony, 
+  Hall, 
+  ChandlerLivingRoom
+]
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+const mystery = {
+  friend: undefined,
+  foods: undefined,
+  rooms: undefined
+}
+
+const pickFriends = () => {
+  mystery.friends = randomSelector(friend)
+  document.getElementById('friendsName').innerHTML = `${mystery.friends.firstName} ${mystery.friends.lastName}`;
+  document.getElementById('friendsAge').innerHTML = `${mystery.friends.age}`;
+  document.getElementById('friendsOccupation').innerHTML = mystery.friends.occupation;
+  document.getElementById('friendsDescription').innerHTML = mystery.friends.description;
+  document.getElementById('friendsImage').src = mystery.friends.image
+}
 
-// This function will be invoked when you click on the killer card.
-const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
+document.getElementById('friendsCard').onclick = pickFriends
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+const pickFoods = () => {
+  mystery.foods = randomSelector(foods)
+  document.getElementById('foodsName').innerHTML = mystery.foods.name;
+  document.getElementById('foodsImage').src = mystery.foods.image
+  document.getElementById('foodsTaste').innerHTML = mystery.foods.taste
 }
 
-// CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
+document.getElementById('foodCard').onclick = pickFoods
+
+const pickRooms = () => {
+  mystery.rooms = randomSelector(rooms)
+  document.getElementById('roomsName').innerHTML = `${mystery.rooms.name}`
+  document.getElementById('roomsImage').src = mystery.rooms.image
+}
 
-// STEP 4 - CREATE A FUNCTION revealMystery that will be invoked when you click that button. It should show something like:
-// 'The murder was committed by Jacob Green, in the living room with a rope.'
+document.getElementById('roomsCard').onclick = pickRooms
+
+const revealMystery = () => {
+  if (mystery.friends === undefined || mystery.foods === undefined || mystery.rooms === undefined ){
+    document.getElementById('mystery').innerHTML = "You must find Joey's food!"
```

Comment:
I like that you put in "You must find Joey's  food!" when I haven't picked any card. That was creative! 😄

Reply:
Thank you Destiny!